go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/cloudflare/circl
 
 go 1.12
 
-require golang.org/x/sys v0.0.0-20190602015325-4c4f7f33c9ed
+require (
+	github.com/mmcloughlin/avo v0.0.0-20200425082757-f891fe8d9749 // indirect
```

Comment:
If you can, you probably want to `avo` being a module dependency for `circl`. It's only a developer tools dependency, not actually a build dependency.

Other `avo` users have achieved this by putting code generators in a separate directory with its own `go.mod` file, for example:

https://github.com/zeebo/blake3/tree/2529747004868782713a339f2f17c8f693db7947/avo

But there's probably other approaches you could take.

Reply:
Thanks for the tip — I almost got it working in a nice way.  I hit the issue that `avo` guesses the package name from the directory it's in.  It would be nice to make that configurable via the commandline.

Reply:
See https://github.com/mmcloughlin/avo/issues/147

Reply:
#112